tools/schemacode/bidsschematools/render.py

```diff
@@ -287,9 +287,32 @@ def make_filename_template(schema, n_dupes_to_combine=6, **kwargs):
     paragraph = paragraph.rstrip()
     codeblock = "Template:\n```Text\n" + paragraph + "\n```"
     codeblock = codeblock.expandtabs(4)
+    codeblock = append_filename_template_legend(codeblock)
     return codeblock
 
 
+def append_filename_template_legend(text=None):
+
+    if text is None:
+        return
+
+    text += """
+
+<details>
+<summary><strong>Legend:</strong></summary>
```

Comment:
Once #1228 is merged, you can use `pdf_format` to toggle the admonition format. That way, if you want to include the legend in the PDF, you can change the formatting, while still keeping the admonition style in the HTML build.